dap_prinz_green_jobs/getters/data_getters.py

```diff
@@ -0,0 +1,186 @@
+from fnmatch import fnmatch
+import json
+import pickle
+import gzip
```

Comment:
fyi @india-kerle I've copied this from ojd_daps_skills

Reply:
I've been using nesta_ds_utils to load and save data from s3! https://github.com/nestauk/nesta_ds_utils 

I still think it would be good to have a data getters for any other loading/saving functions that aren't in nesta_ds_utils though